### PR TITLE
Try/featured image improvements with alt text

### PR DESCRIPTION
**Pull request by anevins12**
## Description
Improved alternative text of featured images to describe the action of the button that wraps it, as well as identifies the featured image.

Branched off PR: #7200

## How has this been tested?
* Passes `npm run test`

## Types of changes
* JavaScript change

## Checklist:
- My code is tested.
- My code follows the WordPress code style. 
- My code follows the accessibility standards. 
- My code has proper inline documentation.

### CODE REVIEW

**Comment by jasmussen** [comment]
Thanks for working on this. Should we continue work in this branch, or should we get #7200 finished and merged first, then rebase this off that?

**Comment by anevins12** [comment]
Yes let's get #7200 finished and merged first :)

File: editor/components/post-featured-image/index.js [hunk 1]
```diff
@@ -41,7 +44,10 @@ function PostFeaturedImage( { featuredImageId, onUpdateImage, onRemoveImage, med
 										naturalWidth={ media.media_details.width }
 										naturalHeight={ media.media_details.height }
 									>
-										<img src={ media.source_url } alt={ __( 'Featured image' ) } />
+										<img
+											alt={ __( 'Replace featured image.' + featuredImageAltText( media.alt_text ) ) }
```
Comment (gziolo) [inline editor/components/post-featured-image/index.js]: This won't work properly, you can't dynamically create a text that is passed to `__()` function. This probably should be rewritten to:

```js
alt={ __( 'Replace featured image.' ) + ' ' + featuredImageAltText( media.alt_text ) }
```

File: editor/components/post-featured-image/index.js [hunk 2]
```diff
@@ -24,6 +24,9 @@ const DEFAULT_REMOVE_FEATURE_IMAGE_LABEL = __( 'Remove featured image' );
 
 function PostFeaturedImage( { featuredImageId, onUpdateImage, onRemoveImage, media, postType } ) {
 	const postLabel = get( postType, [ 'labels' ], {} );
+	const featuredImageAltText = ( altText ) => {
+		return altText ? ' Current image: ' + altText : '';
```
Comment (gziolo) [inline editor/components/post-featured-image/index.js]: This text needs to be wrapped with the translation helper.

**Review by gziolo — DISMISSED** [review]

File: editor/components/post-featured-image/index.js (hunk 2 above)
Comment (anevins12) [reply editor/components/post-featured-image/index.js]: @gziolo  I think I've fixed this, can you check if I've missed something

**Comment by jasmussen** [comment]
@gziolo Thanks for the review, but I think we should get #7200 merged in first, as this PR is essentially an addition to that.

**Comment by gziolo** [comment]
@jasmussen should we land it or close it?

**Comment by jasmussen** [comment]
So, this PR is now so old that I have a hard time compiling it. I tried and got a number of errors with my docker install.

The rephrasing could still make sense, but I think it might be good to see a GIF, or a fresh PR.

**Comment by gziolo** [comment]
@anevins12 would you mind updating this PR with the latest changes from the `master` branch or open a new one as @jasmussen suggested?

**Comment by gziolo** [comment]
Let's close this one and start fresh as @jasmussen suggested.

**Pull request closed without merging**